src/components/Badge/Badge.tsx

```diff
@@ -11,7 +11,7 @@ interface BadgeProps {
     /**
      * Optional Rivet style: an action/error/success/warning badge.
      */
-    variant?: 'action' | 'error' | 'success' | 'warning';
+    variant?: 'action' | 'error' | 'danger' | 'success' | 'warning';
```

Comment:
I think we should go ahead and remove the `error` variant as long as we mention it in the commit message for the release notes.